build/resources.go

```diff
@@ -514,6 +518,18 @@ func createPackagedService(phase string) *corev1.Service {
 	return service
 }
 
+func createPackageConfiguration(webhook webhooks.Webhook, phase string) admissionregv1.ValidatingWebhookConfiguration {
```

Comment:
The should be `createPackagedConfiguration` if we are going to follow the convention for the other func.
But it is better to use  the full name to reduce the confusion? like `createPackagedValidatingWebhookConfigurtation`

Reply:
Updated, Thanks